bundle/LegacyResponse/LegacyResponseManager.php

```diff
@@ -177,7 +178,7 @@ public function mapHeaders(array $headers, Response $response)
                 case 'expires':
                     $response->setExpires(new DateTime($headerValue));
                     break;
-                default;
+                default:
```

Comment:
Came from the fixer automatically (probably by allowing the risky fixers), but since tests passed decided to push it.